9주차/BOJ_10815_숫자카드_라온.py

```diff
@@ -0,0 +1,29 @@
+N = int(input())
+cards = list(map(int, input().split()))
+cards.sort()
```

Comment:
```suggestion
cards = sorted([*map(int, input().split())])
```
이렇게도 된다고 하네요. 저도 5분 전에 배움